### PR TITLE
Default exportThemeId and text mode for multi-select options

### LINKED ISSUE #243: Add exportThemeId
-add exportThemeId on from create to allow forms to be downloaded
-Add the departments select to editor screen, populated with appropriate fields

### LINKED ISSUE #247: Edit multi-select options as text
As a user I'd like to be able to edit options of multi-select fields as text also.
Options would be operated by line break.
e.g. entering:
Selection 1
Selection 2
would convert to two options with corresponding values.

### PR DESCRIPTION
**Pull request by Jdallos**
Resolves #243 
Resolves #247

### CODE REVIEW

File: src/components/editor/editor-screen-drawer.tsx [hunk 1]
```diff
@@ -89,7 +89,8 @@ const EditorScreenDrawer: FC<Props> = ({
       visibility: formSettings.formAuthentication ? MetaformVisibility.Private : MetaformVisibility.Public,
       title: formSettings.formName,
       slug: formSettings.formSlug,
-      sections: formSettings.formSections
+      sections: formSettings.formSections,
+      exportThemeId: "1e7a0019-6a3a-41da-9cb6-6cb01e192ff9"
```
Comment (villejuutila) [inline src/components/editor/editor-screen-drawer.tsx:93]: Actually define this a constant so it it simpler, if it ever needs to be edited.

Comment (Jdallos) [reply src/components/editor/editor-screen-drawer.tsx:93]: done

File: src/components/editor/metaform-editor-right-drawer-feature.tsx [hunk 2]
```diff
@@ -451,6 +455,24 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
     updateFormField(updatedField);
   };
 
+  /**
+   * Event handler for changing edit mode of multi-select type fields
+   *
```
Comment (Jdallos) [inline src/components/editor/metaform-editor-right-drawer-feature.tsx:460]:
```suggestion
   * 
   * @param _event React change event
```

Comment (Jdallos) [reply src/components/editor/metaform-editor-right-drawer-feature.tsx:460]: Add the event param in comments

File: src/components/editor/metaform-editor-right-drawer-feature.tsx [hunk 3]
```diff
@@ -716,21 +738,103 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
...
+   * 
+   * @param event event
+   */
+  const handleMultiSelectTextChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { target: { value } } = event;
+    setMultiSelectRawText(value);
+  };
+
+  /**
+   * Renders multi-choise field raw text edit
```
Comment (Jdallos) [inline src/components/editor/metaform-editor-right-drawer-feature.tsx:781]:
```suggestion
   * Renders multi-choice field raw text edit
```

**Review by Jdallos — COMMENTED** [review]
Couple of changes to comments

File: src/components/editor/metaform-editor-right-drawer-feature.tsx (hunk 2 above)
Comment (villejuutila) [reply src/components/editor/metaform-editor-right-drawer-feature.tsx:460]: Resolved

File: src/components/editor/metaform-editor-right-drawer-feature.tsx (hunk 3 above)
Comment (villejuutila) [reply src/components/editor/metaform-editor-right-drawer-feature.tsx:781]: Resolved

File: src/consts/index.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+  * Api admin user
+  */
+export const API_ADMIN_USER = "api-admin";
+
+/**
+ * Export theme Id
+ */
+export const EXPORT_THEME_ID = "1e7a0019-6a3a-41da-9cb6-6cb01e192ff9";
```
[the remark this answers is not in the document]
Comment (villejuutila) [reply src/consts/index.ts:19]: Resolved

File: src/components/editor/metaform-editor-right-drawer-feature.tsx [hunk 5]
```diff
@@ -716,21 +738,103 @@ const MetaformEditorRightDrawerFeature: FC<Props> = ({
...
-      </Button>
+      <FormControlLabel
+        label={ strings.draftEditorScreen.editor.features.field.addFieldsAsText }
+        control={
+          <Switch
+            checked={ multiSelectRawTextMode }
+            onChange={ handleMultiSelectOptionEditMode }
+          />
+        }
+      />
```
[the remark this answers is not in the document]
Comment (villejuutila) [reply src/components/editor/metaform-editor-right-drawer-feature.tsx:836]: Resolved

**Pull request merged**